openmdao/approximation_schemes/approximation_scheme.py

```diff
@@ -290,6 +291,7 @@ def _compute_approximations(self, system, jac, total, under_cs):
         par_fd_w_serial_model = use_parallel_fd and system._num_par_fd == system._full_comm.size
         num_par_fd = system._num_par_fd if use_parallel_fd else 1
         is_parallel = use_parallel_fd or system.comm.size > 1
+        is_distributed = isinstance(system, Component) and system.options['distributed']
```

Comment:
What happens is we're doing group FD instead of component FD and the group contains distributed components?

Reply:
Group FD is still broken, but this story fixes component FD and check_partials.

Reply:
OK I'll put in a story to fix group FD as well.